server/base/src/main/java/org/apache/accumulo/server/util/fateCommand/FateTxnDetails.java

```diff
@@ -92,6 +93,18 @@ private List<String> formatLockInfo(final List<String> lockInfo,
     return formattedLocks;
   }
 
+  public long getRunning() {
```

Comment:
I'm thinking that the FateTxnDetails variables should be marked final and only initialized in the constructor.